profiles/common/modules/features/nexteuropa_newsroom/nexteuropa_newsroom.admin.inc

```diff
@@ -0,0 +1,597 @@
+    '#collapsible' => TRUE,
+    '#collapsed' => FALSE,
+    '#access' => user_access('administer newsroom advanced settings'),
+  );
+
+  if ($newsroom_universe_id) {
+    $form['newsroom_others']['rebuild'] = array(
+      '#type' => 'submit',
+      '#value' => t('Rebuild importers on the fly'),
+      '#prefix' => '<div id="newsroom_rebuild">',
+      '#suffix' => '<div class="description">' . t("Rebuild the importers if you enabled or disabled languages or if you are noticing unexpected behaviors like fields not imported.") . '</div></div>',
+      '#ajax' => array(
+        'callback' => '_nexteuropa_newsroom_rebuild_importers',
+        'event' => 'click',
+        'wrapper' => 'newsroom_rebuild',
+        'effect' => 'fade',
+        'method' => 'replace',
+      ),
+    );
+  }
+
+  $form['newsroom_others']['clean_cache'] = array(
+    '#type' => 'submit',
+    '#value' => 'Clean newsroom cache',
+    '#description' => t('Clean all newsroom cache entries'),
+    '#prefix' => '<div id="newsroom_clean_cache">',
+    '#suffix' => '<div class="feedback"></div></div>',
+    '#ajax' => array(
+      'callback' => '_nexteuropa_newsroom_clean_cache',
+    ),
+  );
+
+  $form['newsroom_others']['toggle_title'] = array(
+    '#type' => 'submit',
+    '#value' => 'Switch title fields',
+    '#description' => t('Switch title and name fields'),
+    '#prefix' => '<div id="newsroom_toggle_title">',
+    '#suffix' => '<div class="feedback"></div></div>',
+    '#ajax' => array(
+      'callback' => '_nexteuropa_newsroom_toggle_title',
+    ),
+  );
+
+  $form['newsroom_others']['newsroom_order_by_last_update'] = array(
+    '#type' => 'checkbox',
+    '#title' => t('Sort newsroom boxes chronologically in the summary blocks'),
+    '#default_value' => variable_get('newsroom_order_by_last_update', 0),
+  );
+
+  $form['newsroom_others']['newsroom_display_highlights_begin'] = array(
+    '#type' => 'checkbox',
+    '#title' => t('Highlighted topics are displayed at the begining of the list.'),
+    '#default_value' => variable_get('newsroom_display_highlights_begin', 1),
+  );
+
+  $form['newsroom_others']['newsroom_display_agenda'] = array(
+    '#type' => 'checkbox',
+    '#title' => t('Activate agenda functionality'),
+    '#default_value' => variable_get('newsroom_display_agenda', 1),
+  );
+
+  $form['newsroom_others']['newsroom_agenda_after_highlights'] = array(
+    '#type' => 'checkbox',
+    '#title' => t('Display Agenda after the highlights'),
+    '#default_value' => variable_get('newsroom_agenda_after_highlights', 1),
+  );
+
+  $form['newsroom_others']['newsroom_proposal_url'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Newsroom proposal URL'),
+    '#default_value' => variable_get('newsroom_proposal_url', NULL),
+  );
+
+  $vocabulary = taxonomy_vocabulary_machine_name_load(NEXTEUROPA_NEWSROOM_TYPE_VOCABULARY);
```

Comment:
Also should be stored in a static place (drupal variable) because you use the same information with different format in many places, always asking directly database to collect the information what can be staticly stored.